packages/webview_flutter/test/webview_flutter_test.dart

```diff
@@ -115,20 +287,44 @@ class FakePlatformWebView {
 
   MethodChannel channel;
 
-  String lastUrlLoaded;
+  List<String> history = <String>[];
+  int currentPosition = -1;
+  String get lastUrlLoaded =>
```

Comment:
nit: The Flutter style is to only use `=>` if everything fits in one line.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#consider-using--for-short-functions-and-methods